### DISCUSSION TITLE
New T41U5XBB v2.09 setup - spindle output not working ?

### DISCUSSION THREAD

**Discussion by isfl** [phil-barrett/grblHAL-teensy-4.x#93] [Q&A] [answered]
Hello there,
after fixing the motion issue, going further on setting up my spindle output. Using this for an analog laser. I hooked up the 12V supply, getting green LED on power. Running an M3 S100 does nothing to the voltage output, nor PWM pin (my max "rpm" is 100 as I am using a laser). Entering 100 + CW in IoSender does nothing either. Tried inverting Spindle enable pin, flipped spindle setting around, no voltage out of 0-10V pin nor PWM pin. I can disconnect the 12V power, run 100+CW from IOSender and see the red led on, no voltage output on spindle pins. Red led is off if 12V power is hooked up.
Any ideas ?

**Comment by isfl** [comment, marked as answer]
Again, sorted by itself :)
It seems the board had a hardware reset pending and some miscommunication between IOSender and board. Re-connected a few times and both pwm and 0-10v pins work as expected.
Also - unsure if related, but the jumper for relays was off, there was no board config for relays selected.

**Comment by phil-barrett** [comment]
The relay jump only matters if you are driving relay coils.  So, unrelated.

In general, it is a good idea to do a power reset (unplug/replug USB) and restart ioSender after changing any settings.

Also, for laser, you may want to set $30 to 1000. That will give you finer control.  S500 would be 50.0%, S275 would be 27.5%.

**Comment by isfl** [comment]
Thanks Phil ! I don't need the finer control for now, but I'm aware of the power scaling. It makes it simple for us to have max 100 as we use a 100W laser.
